src/rules/newlineBeforeReturnRule.ts

```diff
@@ -53,6 +54,12 @@ class NewlineBeforeReturnWalker extends Lint.AbstractWalker<void> {
         return ts.forEachChild(sourceFile, cb);
     }
 
+    private getIndentation(node: ts.Node): string {
```

Comment:
Seems like this is shared logic between this file, `indentRule.ts`, and `curlyRule.ts`. How about combining their logic into a utils function or two (`src/utils.ts` or similar)? Does that work?